src/MuTalk-Tests/AllTestsMethodsRunningTestSelectionStrategyTest.class.st

```diff
@@ -0,0 +1,36 @@
+Class {
+	#name : 'AllTestsMethodsRunningTestSelectionStrategyTest',
+	#superclass : 'TestSelectionStrategyTest',
+	#category : 'MuTalk-Tests',
+	#package : 'MuTalk-Tests'
+}
+
+{ #category : 'testing' }
+AllTestsMethodsRunningTestSelectionStrategyTest class >> isAbstract [
+	^ false
```

Comment:
Hmm this method is strange... We should review it. There should be no need to say that a class is not abstract